README.markdown

```diff
@@ -8,17 +8,38 @@ Deviating from the guide is allowed, but be prepared to defend your decision in
 In general we prefer clarity over cleverness.
 Try to write code that is easy to understand, debug, and modify.
 
+> [Haskell 98](https://www.haskell.org/onlinereport/) ought to be enough for anybody.
```

Comment:
Should we instead point to https://www.haskell.org/onlinereport/haskell2010/ ?

Reply:
I meant this more as a joke. Maybe it doesn't belong here. We are using Haskell2010 after all. The [things that changed](https://wiki.haskell.org/Haskell_2010#Changes_since_Haskell_.2798) don't really affect the suggestions. Pointing to the most recent version of the standard maintains the spirit of the joke and isn't accidentally confusing.